tensorboard/plugins/distribution/README.md

```diff
@@ -0,0 +1,46 @@
+# The Distribution Dashboard
+
+## Overview
+
+The distributions dashboard lets users
+
+*	Create line charts to display the change in the biases and weights of tensors over time.
+*	Visualize high level statistics, with each line on the graph representing a percentile in the distribution, and the closeness of those lines representing how tightly the variables are being packed into a normal distribution
+
+To use this dashboard, first collect data with summaries and tag it as a histogram, then lay out the UI.
+
+![The Distribution Dashboard](https://user-images.githubusercontent.com/46605868/78409608-168dd100-75d8-11ea-987e-122689d99252.png)
+
+## The problem it solves
+
+The distributions dashboard flattens the display in the histograms dashboard, allowing for easier identification of which percentile of the biases and weights require the most change. The dashboard also includes a transparent overlay of the previous layers results, making it easier to visualize just how much each layer improves results.
+
+## Collecting data
+In order to execute the distributions dashboard, users must first collect [data with summaries](https://www.tensorflow.org/api_docs/python/tf/summary). Both old and new style summaries can be accepted.
```

Comment:
I like that your PR mentions the keras callbacks approach on L23!

> Both old and new style summaries can be accepted.

To be more clear about ways to write data, could we mention/link to the [tf.summary.histogram](https://www.tensorflow.org/api_docs/python/tf/summary/histogram) approach?

e.g. something like
Histogram data may be written from a Python script using [tf.summary.histogram](https://www.tensorflow.org/api_docs/python/tf/summary/histogram).